Sources/TuistGenerator/Generator/Generator.swift

```diff
@@ -65,68 +68,98 @@ public class Generator: Generating {
                                                 configGenerator: configGenerator,
                                                 system: system)
         let environmentLinter = EnvironmentLinter()
-        let workspaceStructureGenerator = WorkspaceStructureGenerator()
         let cocoapodsInteractor = CocoaPodsInteractor()
-        let workspaceGenerator = WorkspaceGenerator(system: system,
-                                                    projectGenerator: projectGenerator,
-                                                    workspaceStructureGenerator: workspaceStructureGenerator,
-                                                    cocoapodsInteractor: cocoapodsInteractor)
+        let workspaceGenerator = WorkspaceGenerator()
```

Comment:
`WorkspaceGenerator` needs to be instantiated with the parameters here otherwise generating individual projects will have different defaults/settings than those generated via the workspace generator